src/integrations/prefect-github/prefect_github/repository.py

```diff
@@ -121,7 +121,7 @@ async def get_directory(
         # Clone to a temporary directory and move the subdirectory over
         with TemporaryDirectory(suffix="prefect") as tmp_dir:
             tmp_path_str = tmp_dir
-            cmd += f" {tmp_path_str}"
+            cmd += f' "{tmp_path_str}"'
```

Comment:
could you either write a test for this change or add a comment above this command linking the relevant issue?